### PR TITLE
optimize: makefile beautify

### PR DESCRIPTION
**Pull request by xxlv**
#### What type of PR is this?
optimize

#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) More detail description for this PR(en: English/zh: Chinese).

en: 
A lightweight Makefile convention is provided, where you can generate descriptions for your Makefile using `##` comments, making it easier to maintain in the future.
zh(optional):
提供了一种轻量的Makefile标记，通过 `##` 注释生成makefile 的描述，后面维护比较方便。

image

### CODE REVIEW

**Comment by welkeyever** [comment]
Perhaps paste the screenshots before and after optimization.

**Comment by welkeyever** [comment]
Also you may check other PRs for how to write description information accurately 🙏

**Comment by xxlv** [comment]
> Also you may check other PRs for how to write description information accurately 🙏

Ok

**Comment by welkeyever** [comment]
One of the commit has not signed the CLA(innovation-2022), PTAL.

File: Makefile [hunk 1]
```diff
@@ -8,39 +8,28 @@ SHELL := /bin/bash
...
-	@echo ''
-	@echo 'Targets run by default are: lint, vet.'
-	@echo ''
+all: imports fmt lint vet errors build
 
 print-%:
 	@echo $* = $($*)
 
-deps:
+deps:
```
Comment (welkeyever) [inline Makefile:20]: Maybe add this to help menu also?

**Comment by welkeyever** [comment]
@xxlv  Hi~ Is there any update for this pr?

**Pull request closed without merging**